jax_galsim/wcs.py

```diff
@@ -0,0 +1,922 @@
+        return self._x(u, v), self._y(u, v)
+
+    # For LocalWCS, this is of course trivial.
+    def _local(self, image_pos, color):
+        return self
+
+
+#########################################################################################
+#
+# Local WCS classes are those where (x,y) = (0,0) corresponds to (u,v) = (0,0).
+#
+# We have the following local WCS classes:
+#
+#     PixelScale
+#     ShearWCS
+#     JacobianWCS
+#
+# They must define the following:
+#
+#     origin            attribute or property returning the origin
+#     world_origin      attribute or property returning the world origin
+#     _u                function returning u(x,y)
+#     _v                function returning v(x,y)
+#     _x                function returning x(u,v)
+#     _y                function returning y(u,v)
+#     _profileToWorld   function converting image_profile to world_profile
+#     _profileToImage   function converting world_profile to image_profile
+#     _pixelArea        function returning the pixel area
+#     _minScale         function returning the minimum linear pixel scale
+#     _maxScale         function returning the maximum linear pixel scale
+#     _toJacobian       function returning an equivalent JacobianWCS
+#     _writeHeader      function that writes the WCS to a fits header.
+#     _readHeader       static function that reads the WCS from a fits header.
+#     _newOrigin        function returning a non-local WCS corresponding to this WCS
+#     copy              return a copy
+#     __eq__            check if this equals another WCS
+#     __repr__          convert to string
+#
+#########################################################################################
+
+
+@_wraps(_galsim.PixelScale)
+@register_pytree_node_class
+class PixelScale(LocalWCS):
+    def __init__(self, scale):
+        self._params = {"scale": scale}
+        self._scale = jnp.asarray(scale)
+        self._color = None
+
+    # Help make sure PixelScale is read-only.
+    @property
+    def scale(self):
+        """The pixel scale"""
+        return self._scale
+
+    def isPixelScale(self):
+        return True
+
+    def _u(self, x, y, color=None):
+        return x * self._scale
+
+    def _v(self, x, y, color=None):
+        return y * self._scale
+
+    def _x(self, u, v, color=None):
+        return u / self._scale
+
+    def _y(self, u, v, color=None):
+        return v / self._scale
+
+    def _profileToWorld(self, image_profile, flux_ratio, offset):
+        return _Transform(
+            image_profile,
+            (self._scale, 0.0, 0.0, self._scale),
+            flux_ratio=self._scale**-2 * flux_ratio,
+            offset=offset,
+        )
+
+    def _profileToImage(self, world_profile, flux_ratio, offset):
+        return _Transform(
+            world_profile,
+            (1.0 / self._scale, 0.0, 0.0, 1.0 / self._scale),
+            flux_ratio=self._scale**2 * flux_ratio,
+            offset=offset,
+        )
+
```

Comment:
there are some Shear methods missing here, I assume it's because we don't have shear objects. But why not just include it so we don't forget later? (or at least leave #TODO)

Reply:
`_shearToWorld` and `_shearToImage`